packages/wrapped-keys/src/lib/litActions/solana/src/signMessageWithSolanaEncryptedKey.js

```diff
@@ -0,0 +1,70 @@
+const { Keypair } = require('@solana/web3.js');
+const bs58 = require('bs58');
+const nacl = require('tweetnacl');
+
+(async () => {
+  const LIT_PREFIX = 'lit_';
+
```

Comment:
Let's add some check for user params passes in the jsPrams like in `signTransactionWithEthereumEncryptedKeyLitAction`

Reply:
The user params required are:
```
jsParams: {
      accessControlConditions,
      ciphertext,
      dataToEncryptHash,
      messageToSign,
    },
```
do you want me to check for all of these? The `signMessageWithEthereumEncryptedKeyLitAction` doesn't check for any of them, should it be updated to?

Reply:
Makes sense as these will be added internally when calling the function `signMessageWithEncryptedKey`. That's why we should use these functions as the functionality is already present for both Eth & Solana